en/DatabaseProperties.md

```diff
@@ -1,28 +1,60 @@
 ---
 title: Database properties window
-outdated: true
 ---
 
 # Database properties window
 
-*Opened from the main window by selecting **File -&gt; Database properties***
+Each database can have specific properties that can be modified through **File -&gt; Database Properties**).
+These specific properties override the default properties (which are defined in **Options -&gt; Preferences**).
```

Comment:
I'm wondering if "default" is the best word here? I cannot really come up with a better wording though. The default properties are more likely the ones obtained by resetting the properties.

"These specific properties override the generic properties defined in **Options -&gt; Preferences**."?